pgdrive/envs/pgdrive_env.py

```diff
@@ -812,11 +597,19 @@ def for_each_vehicle(self, func, *args, **kwargs):
     def vehicle(self):
         """A helper to return the vehicle only in the single-agent environment!"""
         assert len(self.vehicles) == 1, "env.vehicle is only supported in single-agent environment!"
-        return self.vehicles[DEFAULT_AGENT]
+        ego_v = self.vehicles[DEFAULT_AGENT]
+        return ego_v
 
     def reward(self, *args, **kwargs):
         raise ValueError("reward function is deprecated!")
 
+    @property
+    def use_image(self):
```

Comment:
The name is confusing, change to use_render or use_camera or use_first_view or so.